packages/plugin-patch/sources/commands/patch.ts

```diff
@@ -14,7 +14,7 @@ export default class PatchCommand extends BaseCommand {
   static usage: Usage = Command.Usage({
     description: `prepare a package for patching`,
     details: `
-      This command will cause a package to be extracted in a temporary directory (under a folder named "patch-workdir"). This folder will be editable at will; running \`yarn patch\` inside it will then cause Yarn to generate a patchfile and register it into your top-level manifest (cf the \`patch:\` protocol).
+      This command will cause a package to be extracted in a temporary directory. This folder will be editable at will; once it's ready, run \`yarn patch-commit -s <path>\`, which will then cause Yarn to generate a patchfile (in \`.yarn/patch\`) and register it into your top-level manifest (cf the [\`patch:\` protocol](https://yarnpkg.com/features/protocols#patch)).
```

Comment:
```suggestion
      This command will cause a package to be extracted in a temporary directory. This folder will be editable at will; once it's ready, run \`yarn patch-commit -s <path>\`, which will then cause Yarn to generate a patchfile (in \`.yarn/patches\`) and register it into your top-level manifest (cf the [\`patch:\` protocol](https://yarnpkg.com/features/protocols#patch)).
```
The path is configurable so this might not be the true path